src/utils/hooks/api/loans/use-get-loans-data.tsx

```diff
@@ -0,0 +1,61 @@
+import { CurrencyIdLiteral } from '@interlay/interbtc-api';
```

Comment:
What's the timescale on getting mock data returned from the lib? I'd prefer to keep mock data out of the UI (even if it's only temporary) but depends whether this is a blocker for you continuing work.